main.go

```diff
@@ -109,6 +109,7 @@ func main() {
 		config.WithVersion(v),
 		config.WithCollectorImage(collectorImage),
 		config.WithTargetAllocatorImage(targetAllocatorImage),
+		config.WithAutoInstrumentationJavaImage(autoInstrumentationJava),
```

Comment:
@pavolloffay Curious here, Why does Java auto-instrumentation logic is part of `main.go` but not the nodejs and python instrumentation logic? I'm assuming the auto-instrumentation is done using the sidecar so mostly the logic of injecting the container using the mutating webhook should be common. 

Note: I haven't followed the instrumentation PRs closely. I will make some time to go through the instrumentation pkg.

Reply:
great point :)))

> but not the nodejs and python instrumentation logic?

because the nodejs and pythond does not support defaulting for the image and upgrade. Once this is merged I will submit a PR to make all auto-instrumentation the same.